SPECS/tpm2-tools/tpm2-tools.spec

```diff
@@ -1,20 +1,27 @@
 Summary:        The source repository for the TPM (Trusted Platform Module) 2 tools
 Name:           tpm2-tools
-Version:        4.3.2
+Version:        5.5
 Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://github.com/tpm2-software/tpm2-tools
-Source0:        https://github.com/tpm2-software/tpm2-tools/releases/download/%{version}/%{name}-%{version}.tar.gz
-BuildRequires:  curl-devel
-BuildRequires:  openssl-devel
-BuildRequires:  tpm2-tss-devel >= 2.3.0
-Requires:       /bin/awk
-Requires:       curl
-Requires:       openssl
-Requires:       tpm2-tss >= 2.3.0
+
+Source0: https://github.com/tpm2-software/tpm2-tools/releases/download/%{version}/%{name}-%{version}.tar.gz
+%define sha512 %{name}=224a5ea3448a877362abb35ac06b115c559c09b44b30d74c8326211be66d24e0e130c285b1e285be1842e7203ab488629b0f4e451cbd782c83ed72023d146675
```

Comment:
this is the sha of the latest version from *Photon*, which is 5.3. But you are including version *5.5* here, which has a different hash. So either you should remove this hash line completely (I don't see the point of it) or update it with the actual correct hash.

Reply:
will remove it...thanks!